README.md

```diff
@@ -128,284 +43,8 @@
-  def inspect(true),  do: "true"
-  def inspect(nil),   do: "nil"
-
-  # Good
-  module = env.module
-  arity = length(args)
-
-  def inspect(false), do: "false"
-  def inspect(true), do: "true"
-  def inspect(nil), do: "nil"
-  ```
-
-  The same non-alignment rule applies to `<-` and `->` clauses as well.
-
-* <a name="pipeline-indentation"></a>
-  Use a single level of indentation for multi-line pipelines.
-  <sup>[[link](#pipeline-indentation)]</sup>
-
-  ```elixir
-  input
-  |> String.strip()
-  |> String.downcase()
-  |> String.slice(1, 3)
-  ```
-
-### Term representation
-
-* <a name="underscores-in-numerics"></a>
-  Add underscores to decimal literals that have six or more digits.
-  <sup>[[link](#underscores-in-numerics)]</sup>
-
-  ```elixir
-  # Bad
-  num = 1000000
-  num = 1_500
-
-  # Good
-  num = 1_000_000
-  num = 1500
-  ```
-
-* <a name="hex-literals"></a>
-  Use uppercase letters when using hex literals.
-  <sup>[[link](#hex-literals)]</sup>
-
-  ```elixir
-  # Bad
-  <<0xef, 0xbb, 0xbf>>
-
-  # Good
-  <<0xEF, 0xBB, 0xBF>>
-  ```
-
-* <a name="quotes-around-atoms"></a>
-  When using atom literals that need to be quoted because they contain characters that are invalid in atoms (such as `:"foo-bar"`), use double quotes around the atom name:
-  <sup>[[link](#quotes-around-atoms)]</sup>
-
-  ```elixir
-  # Bad
-  :'foo-bar'
-  :'atom number #{index}'
-
-  # Good
-  :"foo-bar"
-  :"atom number #{index}"
-  ```
-
-* <a name="trailing-comma"></a>
-  When dealing with lists, maps, structs, or tuples whose elements span over multiple lines and are on separate lines with regard to the enclosing brackets, it's advised to use a trailing comma even for the last element:
-  <sup>[[link](#trailing-comma)]</sup>
-
-  ```elixir
-  [
-    :foo,
-    :bar,
-    :baz,
-  ]
-  ```
-
 ### Parentheses
 
-* <a name="zero-arity-parens"></a>
-  Parentheses are a must for __local__ or __imported__ zero-arity function calls.
-  <sup>[[link](#zero-arity-parens)]</sup>
-
-  ```elixir
-  # Bad
-  pid = self
-  import System, only: [schedu]
-
-  # Good
-  pid = self()
-  def new(), do: %MapSet{}
-  config = IEx.Config.new
-  ```
-
-  The same should be done for __remote__ zero-arity function calls:
-
-  ```elixir
-  # Bad
-  Mix.env
-
-  # Good
-  Mix.env()
-  ```
-
-  This rule also applies to one-arity function calls (both local and remote) in pipelines:
-
-  ```elixir
-  # Bad
-  input
-  |> String.strip
-  |> decode
-
-  # Good
-  input
-  |> String.strip()
-  |> decode()
-  ```
-
-* <a name="anonymous-fun-parens"></a>
-  Never wrap the arguments of anonymous functions in parentheses.
-  <sup>[[link](#anonymous-fun-parens)]</sup>
-
-  ```elixir
-  # Bad
-  Agent.get(pid, fn(state) -> state end)
-  Enum.reduce(numbers, fn(number, acc) ->
-    acc + number
-  end)
-
-  # Good
-  Agent.get(pid, fn state -> state end)
-  Enum.reduce(numbers, fn number, acc ->
-    acc + number
-  end)
-  ```
-
 * <a name="fun-parens"></a>
```

Comment:
We can't enforce this one entirely as changing `def one do` to `def one() do` is a semantic change.